src/components/Drawer/Drawer.tsx

```diff
@@ -20,16 +20,32 @@ export interface DrawerProps extends React.HTMLAttributes<HTMLDivElement> {
    * The content of the drawer
    */
   children?: React.ReactNode;
+  /**
+   * Drawer optional variant
+   */
+  variant?: 'bottomSheet' | 'default';
+  /**
+   * Option drawer open side
+   */
+  drawerOpenSide?: 'left' | 'right';
 }
 /**
  * ## Overview
  *
  * A component for displaying a drawer.
  *
  */
-const Drawer = ({ className, isOpen = false, onClose = noOp, children, ...props }: DrawerProps) => {
-  const { className: baseClassName, ...commonProps } = getCommonProps(props, 'Drawer');
-
+const Drawer = ({
+  className,
+  isOpen = false,
+  onClose = noOp,
+  children,
+  variant = 'default',
+  drawerOpenSide = 'right',
+  ...props
+}: DrawerProps) => {
+  const isBottomSheet = variant === 'bottomSheet';
+  const { className: baseClassName, ...commonProps } = getCommonProps(props, isBottomSheet ? 'BottomSheet' : 'Drawer');
```

Comment:
If possible can we keep the same `Drawer` `baseClassName` and remove the `BottomSheet` version and just adjust our scss classnames accordingly